db/migrate/20170706172353_remove_category_column_from_repositories.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCategoryColumnFromRepositories < ActiveRecord::Migration
+  def change
+    remove_column :repositories, :category, :string
```

Comment:
This needs to be in a separate PR since you have code in here that depends on this